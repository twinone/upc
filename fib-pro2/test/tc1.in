rellotge 10:30
+ 10:29 error
rellotge 10:29
sortir
