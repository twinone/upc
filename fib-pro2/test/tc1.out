No s'ha realitzat
No s'ha realitzat
